sdk/identity/azure-identity/src/token_credential_impl.cpp

```diff
@@ -192,13 +194,39 @@ std::string TokenAsDiagnosticString(
       + "\' property.\nSee Azure::Core::Diagnostics::Logger for details"
         " (https://aka.ms/azsdk/cpp/identity/troubleshooting).");
 }
+
+std::string TimeZoneOffsetAsString(int utcDiffSeconds)
+{
+  if (utcDiffSeconds == 0)
+  {
+    return {};
+  }
+
+  std::ostringstream os;
+  if (utcDiffSeconds >= 0)
+  {
+    os << '+';
+  }
```

Comment:
nit: It should never be == 0 at this point, right?
```suggestion
  if (utcDiffSeconds > 0)
  {
    os << '+';
  }
```

Unless, you think `+00:00` is a safer choice than `-00:00` which would be the "else" case, otherwise.

Reply:
We should never get `== 0` here, but should we ever get here, I think, `+00:00` is aesthetically better.